box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -2,33 +2,36 @@
 
 import android.app.ProgressDialog;
 import android.content.Intent;
-import android.support.v7.app.ActionBarActivity;
+import android.graphics.Typeface;
 import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.view.Menu;
+import android.view.MenuItem;
 import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
 import android.widget.Toast;
 
 import com.box.androidsdk.content.BoxApiFolder;
 import com.box.androidsdk.content.BoxConfig;
-import com.box.androidsdk.content.models.BoxIteratorCollaborations;
-import com.box.androidsdk.content.models.BoxIteratorItems;
-import com.box.androidsdk.content.models.BoxSession;
 import com.box.androidsdk.content.BoxException;
 import com.box.androidsdk.content.models.BoxEntity;
 import com.box.androidsdk.content.models.BoxError;
 import com.box.androidsdk.content.models.BoxFolder;
 import com.box.androidsdk.content.models.BoxItem;
 import com.box.androidsdk.content.models.BoxIteratorCollaborations;
 import com.box.androidsdk.content.models.BoxIteratorItems;
+import com.box.androidsdk.content.models.BoxSession;
 import com.box.androidsdk.content.models.BoxSharedLink;
-import com.box.androidsdk.share.CollaborationUtils;
 import com.box.androidsdk.share.activities.BoxActivity;
-import com.box.androidsdk.share.activities.BoxCollaborationsActivity;
 import com.box.androidsdk.share.activities.BoxInviteCollaboratorsActivity;
 import com.box.androidsdk.share.activities.BoxSharedLinkActivity;
 
 import java.net.HttpURLConnection;
 import java.util.ArrayList;
 
+import static com.box.androidsdk.sample.R.id.launchCollabs;
```

Comment:
I was quite surprised to look at that as well. I think I accidentally forgot to fully reference the resource object so in turn IntelliJ automatically added the import statement (because of my settings in Preferences). I made the mistake in line 81. Good catch! I will fix it.